src/Smalot/PdfParser/RawData/RawDataParser.php

```diff
@@ -455,59 +455,67 @@ protected function decodeXrefStream($pdfData, $startxref, $xref = [])
      *
      * @param string $pdfData  PDF data
      * @param array  $xref
-     * @param string $obj_ref  Object number and generation number separated by underscore character
+     * @param string $objRef   Object number and generation number separated by underscore character
      * @param int    $offset   Object offset
      * @param bool   $decoding If true decode streams
      *
      * @return array containing object data
      *
      * @throws Exception if invalid object reference found
      */
-    protected function getIndirectObject($pdfData, $xref, $obj_ref, $offset = 0, $decoding = true)
+    protected function getIndirectObject($pdfData, $xref, $objRef, $offset = 0, $decoding = true)
     {
-        $obj = explode('_', $obj_ref);
-        if (2 != \count($obj)) {
+        // --- build indirect object header
+        // $objHeader = "[object number] [generation number] obj"
```

Comment:
Please remove code which is commented out.

Reply:
this is more like pseudo code to know what is the result of the next few lines